src/lm_robust_helper.cpp

```diff
@@ -35,7 +35,7 @@ List lm_solver(Eigen::Map<Eigen::MatrixXd>& Xfull,
   // Much of the OLS solution code is inspired by or copied directly from the fastLm function from RcppEigen
   // https://cran.r-project.org/web/packages/RcppEigen/vignettes/RcppEigen-Introduction.pdf
   try {
-    if (trychol) {
+    if (try_cholesky) {
```

Comment:
Catching your own exception as below is a bit gross - should probably break this in to smaller functions at some point.

Reply:
I agree. I'll return this when I sweep through again.